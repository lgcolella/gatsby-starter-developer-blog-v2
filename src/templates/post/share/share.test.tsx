/* Vendor imports */
import React from 'react'
import { shallow } from 'enzyme'
/* App imports */
import Share from './index'

describe('Post > Share component', () => {
  test('matches snapshot', () => {
    let wrapper = shallow(
      <Share
        pageCanonicalUrl={'/path/canonical-url'}
        title={'Title of the page'}
      />
    )

    expect(wrapper).toMatchSnapshot()
  })
})
