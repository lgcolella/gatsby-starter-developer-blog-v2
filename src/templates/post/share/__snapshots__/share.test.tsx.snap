// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post > Share component matches snapshot 1`] = `
<div>
  <div
    className="container"
  >
    <ForwardRef
      url="/path/canonical-url"
      windowHeight={400}
      windowWidth={550}
    >
      <Chip
        className="chip"
        hoverEffect={true}
      >
        <span>
          <FaFacebookF
            className="icon"
          />
          Facebook
        </span>
      </Chip>
    </ForwardRef>
    <ForwardRef
      title="Title of the page"
      url="/path/canonical-url"
      windowHeight={400}
      windowWidth={550}
    >
      <Chip
        className="chip"
        hoverEffect={true}
      >
        <span>
          <FaTwitter
            className="icon"
          />
          Twitter
        </span>
      </Chip>
    </ForwardRef>
    <ForwardRef
      url="/path/canonical-url"
      windowHeight={600}
      windowWidth={750}
    >
      <Chip
        className="chip"
        hoverEffect={true}
      >
        <span>
          <FaLinkedinIn
            className="icon"
          />
          LinkedIn
        </span>
      </Chip>
    </ForwardRef>
    <ForwardRef
      title="Title of the page"
      url="/path/canonical-url"
      windowHeight={460}
      windowWidth={660}
    >
      <Chip
        className="chip"
        hoverEffect={true}
      >
        <span>
          <FaRedditAlien
            className="icon"
          />
          Reddit
        </span>
      </Chip>
    </ForwardRef>
    <ForwardRef
      onClick={[Function]}
      openWindow={false}
      separator=" "
      subject="Title of the page"
      url="/path/canonical-url"
    >
      <Chip
        className="chip"
        hoverEffect={true}
      >
        <span>
          <FaEnvelope
            className="icon"
          />
          Email
        </span>
      </Chip>
    </ForwardRef>
  </div>
</div>
`;
